release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -240,6 +240,16 @@ For more information, see xref:../edge_computing/cnf-talm-for-cluster-upgrades.a
 
 SHA1 certificates are no longer supported for use with HaProxy. Both existing and new routes using SHA1 certificates in {product-version} are rejected and no longer function.
 
+[discrete]
+[id="ocp-4-16-etcd-tuning-profiles"]
+=== etcd tuning parameters
+
+With this release, the etcd tuning parameters can be set to values that optimize performance and decrease latency, as follows.
+
+* "" (Default)
+* Standard
+* Slower
```

Comment:
@tmalove: I think these parameters should be in backticks, which is what you had for the 4.15 TP docs: https://docs.openshift.com/container-platform/4.15/scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.html#etcd-tuning-parameters_recommended-etcd-practices

```suggestion
* `""` (Default)
* `Standard`
* `Slower`
```
Let me know what you think and feel free to DM me when you are ready for a merge.

/remove-label merge-review-in-progress
/remove-label merge-review-needed